feathr_project/feathr/api/app/backends/empty_backend_impl.py

```diff
@@ -0,0 +1,100 @@
+from app.backends.backend_base import FeathrBackendAbstract
+from app.core.error_code import ErrorCode
+from app.core.feathr_api_exception import FeathrApiException
+
+
+class FakeBackend(FeathrBackendAbstract):
```

Comment:
What's this for?

Reply:
Seems not need to use this for now, will be deteled